metrics/pipe_analysis.yaml

```diff
@@ -0,0 +1,17 @@
+# Metric definitions in pipe_analysis
+
+stellar_w_principal_color_rms_width_gri:
+  description: >
+    The width of the "w" principle color stellar locus: the blue section on the r-i vs g-r plane
+    using PSF fluxes.
+  unit: mmag
+  reference:
+    doc: Ivezic, Z. et al. 2004, AN 325, SDSS data management and photometric quality assessment
+    url: https://onlinelibrary.wiley.com/doi/pdf/10.1002/asna.200410285
+    page: 586
+  tags:
+      - photometry
+      - stellarLocus
+      - coadd
+      - PSFflux
+      - colorAnalysisTask
```

Comment:
This looks good, keep in mind that in the future these tags might be used to filter or group metrics when you make your queries in Chronograf.